roles/servicemesh/tasks/main.yml

```diff
@@ -0,0 +1,343 @@
+---
+- name: Get current state
+  k8s_info:
+    api_version: ripsaw.cloudbulldozer.io/v1alpha1
+    kind: Benchmark
+    name: '{{ meta.name }}'
+    namespace: '{{ operator_namespace }}'
+  register: resource_state
+
+- name: Define variables
+  set_fact:
+    servicemesh: "{{ servicemesh_defaults | combine(workload_args, recursive=True) }}"
+    controlplane_namespace: "{{ meta.name }}-controlplane-{{ trunc_uuid }}"
+    workload_namespace: "{{ meta.name }}-workload-{{ trunc_uuid }}"
+    owner:
+      - apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+        kind: Benchmark
+        name: "{{ meta.name }}"
+        namespace: "{{ operator_namespace }}"
+        uid: "{{ resource_state.resources[0].metadata.uid }}"
+
+- when: resource_state.resources[0].status.state is not defined
+  block:
+  - name: Create control plane namespace
```

Comment:
Can we just have this be something the user provides? `ie in the CR provide us with the namespace for the control plane? Ensure it has this as part of the RBAC?`

Reply:
Reading through your commit again, it might be necessary for service mesh... Maybe we can have you demo this to us, this week @rvansa ?

Reply:
Again, that would be another step backwards from automation.

Regarding a demo, this week I don't have OCP installation available (our scalelab allocation has expired and we're in the process of reinstalling OCP in our own lab). Have you tried to install SM and HF operators from the catalog and just start the benchmark?